### PR TITLE
Feature/ms3 11 adapt binary transformation

### PR DESCRIPTION
**Pull request by rhoxhaj**
## Abstract
when this endpoint is called with a s3binary field:

- Mesh will download the binary (in the temp folder)
- apply transformations (same as existing binaryTransformation) and return json with updated information.

### CODE REVIEW

File: rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryFieldTransformRequest.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+import com.gentics.mesh.core.rest.common.RestModel;
+import com.gentics.mesh.core.rest.node.field.image.FocalPoint;
+import com.gentics.mesh.parameter.image.CropMode;
+import com.gentics.mesh.parameter.image.ImageRect;
+import com.gentics.mesh.parameter.image.ResizeMode;
+
+/**
+ * POJO for a binary field transform request
+ */
+public class S3BinaryFieldTransformRequest implements RestModel {
```
Comment (plyhun) [inline rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryFieldTransformRequest.java:14]: Can't we reuse the existing BinaryFieldTransformRequest, at it looks like identical?
Need also your opinions on this @bernhardriegler , @yrucrem .

Comment (rhoxhaj) [reply rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryFieldTransformRequest.java:14]: Actually I agree with you @plyhun . When i started working on it, I planned there would be some change in the functionality, which didn't happen. I would opt for the usage of BinaryFieldTransformRequest.

Comment (bernhardriegler) [reply rest-model/src/main/java/com/gentics/mesh/core/rest/node/field/S3BinaryFieldTransformRequest.java:14]: @plyhun good suggestion - @rhoxhaj please go ahead and use the existing BinaryFieldTransformRequest

File: mdm/common/src/main/java/com/gentics/mesh/core/image/ImageManipulator.java [hunk 2]
```diff
@@ -47,38 +47,36 @@
     Single<File> handleS3Resize(String bucketName, String s3ObjectKey, String filename, ImageManipulationParameters parameters);
```
Comment (plyhun) [inline mdm/common/src/main/java/com/gentics/mesh/core/image/ImageManipulator.java:47]: Here it looks like only the indentation changed, so at least this file can be reverted.

Comment (plyhun) [reply mdm/common/src/main/java/com/gentics/mesh/core/image/ImageManipulator.java:47]: UPD: caught the old commit, all ok in the end.

**Comment by plyhun** [comment]
Looks ok to me.

**Pull request merged**